common/list/_lists.scss

```diff
@@ -38,18 +41,13 @@
 /* Double space list items
 	Might worth to publish it upstream here instead: https://github.com/wet-boew/wet-boew/blob/master/src/base/lists/_base.scss#L30
  */
-
+div.lst-spcd-2>ul,
 .lst-spcd-2 {
 	> {
 		li {
 			margin-bottom: 20px;
 		}
 	}
-
-	ul,
-	ol {
-		margin-top: 20px;
-	}
```

Comment:
This is removed? 

It will need to be put it back. Feel free to reorganize the CSS or to duplicate the current one for your edge case specifically

Reply:
Fixed